docs/src/features.md

```diff
@@ -170,6 +170,12 @@ Inlay hints are disabled by default and can be enabled with the `"show_inlay_hin
 
 !!! info "Some servers require additional settings to be enabled in order to show inlay hints."
 
+## Inline Completions
+
+Inline completions are typically provided by an AI code assistant.
+They can span multiple lines and are rendered directly in the source code as grayed out text ("ghost text").
+Currently inline completions are only requested when you manually trigger auto-completions (<kbd>Ctrl</kbd> + <kbd>Space</kbd>).
+
```

Comment:
I would suggest adding here that Inline completions will not be triggered if `mini_auto_complete` is enabled.